CHANGELOG

```diff
@@ -0,0 +1,22 @@
+# Changelog
+
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
+
+## [Unreleased]
+
+### Fixes 
```

Comment:
@cpytel The official format of _Keep a Changelog_ defines **Fixed** not ~Fixes~.

```diff
- ### Fixes
+ ### Fixed
```

Reply:
Thanks for catching it! Fixed in https://github.com/thoughtbot/laptop/commit/394348eb925e4509e9e21728858762f698ebe005